src/services/company/company.email.service.ts

```diff
@@ -17,17 +17,15 @@ export const getCompanyEmail = async (companyNumber: string): Promise<Registered
   const sdkResponse: Resource<RegisteredEmailAddress> = await privateApiClient.registeredEmailAddress.getRegisteredEmailAddress(companyNumber);
 
   if (!sdkResponse) {
-    throw createAndLogError( SERVICE_UNAVAILABLE, `Registered email address API for company number ${companyNumber}`);
+    throw createAndLogError( THERE_IS_A_PROBLEM_ERROR, `Registered email address API for company number ${companyNumber}`);
   }
 
-  if (sdkResponse.httpStatusCode === StatusCodes.SERVICE_UNAVAILABLE || sdkResponse.httpStatusCode === StatusCodes.INTERNAL_SERVER_ERROR) {
-    throw createAndLogError( SERVICE_UNAVAILABLE, `Registered email address API for company number ${companyNumber}`);
-  } else if (sdkResponse.httpStatusCode >= StatusCodes.BAD_REQUEST) {
-    throw createAndLogError( SOMETHING_HAS_GONE_WRONG, `Http status code ${sdkResponse.httpStatusCode} - Failed to get registered email address for company number ${companyNumber}`);
+  if (sdkResponse.httpStatusCode >= StatusCodes.BAD_REQUEST) {
```

Comment:
This doesn't look quite right maybe something like `!= StatusCodes.OK` would work better

Reply:
Agreed